buffer.c

```diff
@@ -3080,7 +3080,11 @@ evbuffer_file_segment_materialize(struct evbuffer_file_segment *seg)
 			offset_leftover = offset % page_size;
 			offset_rounded = offset - offset_leftover;
 		}
+#if defined(EVENT__HAVE_MMAP64)
+		mapped = mmap64(NULL, length + offset_leftover,
```

Comment:
Hm, in which env (libc, os, kernel version) did you faced with an issue that you were not able to mmap 2GB+ file? On x86?

Reply:
By they way, seems that `*64` should not be used directly and instead `_FILE_OFFSET_BITS` should be used?

Reply:
I had issues with 2Gb+ files on Linux 4.9.37 with glibc version. But it turns out that `_FILE_OFFSET_BITS` is specific only for glibc and will not work with musl

Reply:
>I had issues with 2Gb+ files on Linux 4.9.37 with glibc version

On x86 not x86_64, is this correct?

Reply:
on ARM32